docker-compose.yml

```diff
@@ -199,3 +221,4 @@ services:
       - OTEL_RESOURCE_ATTRIBUTES=service.name=loadgenerator
     depends_on:
       - frontend
+      - otelcol
```

Comment:
The `loadgenerator` only depends on the `frontend`

Reply:
ok